python/requirements/ml/requirements_dl.txt

```diff
@@ -11,7 +12,7 @@ tensorflow-probability==0.19.0
 torch==1.13.0
 torchvision==0.14.0
 torch-scatter==2.1.0
-torch-sparse==0.6.15
+torch-sparse==0.6.16
```

Comment:
for correctness also needs updating here: https://github.com/ray-project/ray/blob/ray-2.3.0/python/requirements/ml/requirements_ml_docker.txt#L14